borb/pdf/canvas/layout/page_layout/multi_column_layout.py

```diff
@@ -76,7 +77,9 @@ def __init__(
             self._vertical_margin = vertical_margin
 
         # inter-column margin
-        self._inter_column_margin = self._page_width * Decimal(0.05)
+        if gutter_width is None:
+            gutter_width = 0.05
+        self._inter_column_margin = self._page_width * Decimal(gutter_width)
```

Comment:
If you would like to expose this parameter to the outside world, it needs to be clear what it does.
i.e. it needs to be clear whether the `gutter_width` is fixed or multiplicative.
You also need to add tests that showcase different values.
You also need asserts to ensure nobody can enter a value that would cause the layout system to crash.